From 95fe2712df38769037cafce2e7fc9647e6cee068 Mon Sep 17 00:00:00 2001
From: Bernhard Rosenkraenzer <bero@arklinux.org>
Date: Tue, 24 May 2011 15:35:58 +0159
Subject: [PATCH 08/23] Fix start page not being valid XML (but claiming to be
 XHTML) Broke displaying the start page in current WebKit snapshots

---
 src/htmls/startpage.html | 2 ++
 1 file changed, 2 insertions(+)

diff --git src/htmls/startpage.html src/htmls/startpage.html
index 94d53a0..d5f6731 100644
--- src/htmls/startpage.html
+++ src/htmls/startpage.html
@@ -6,6 +6,7 @@
     <title></title>
     <link rel="stylesheet" href="qrc:/startpage.css" type="text/css" />
     <script type="text/javascript">
+        //<![CDATA[
         function update()
         {
             var searchEngine = window.arora.currentEngine;
@@ -40,6 +41,7 @@
             var url = window.arora.searchUrl(string);
             window.location.href = url;
         }
+        //]]>
     </script>
 </head>
 <body onload="document.forms[0].lineEdit.select(); update();">
-- 
2.44.0

